tests/datasets/test_datasets.py

```diff
@@ -91,3 +92,24 @@ def test_demo_replay_dataset():
     for _, actions in dataset:
         for action in actions:
             assert not action.has_option()
+
+
+def test_teacher_dataset():
+    """Test teacher dataset creation with Covers env.
+    """
+    # Test that data does not contain options since approach is random
+    utils.update_config({
+        "env": "cover",
+        "approach": "random",
+        "offline_data_method": "demo",
+        "offline_data_planning_timeout": 500,
+        "seed": 0,
+        "num_train_tasks": 7,
+    })
+    env = CoverEnv()
+    dataset = create_dataset(env)
```

Comment:
Can you add a test that checks if the new hyperparameter is being used as expected? The easiest thing would be to just assert that the length of the ground atoms in one of the examples is equal to a magic number. Then we'll be alerted in the future if something changes unexpectedly. A more robust test would recompute all groundatoms for a given state and check that the number of groundatoms in the dataset is proportional to the expected frac.